src/styles/blocks/rating.css

```diff
@@ -0,0 +1,44 @@
+.rating {
+  display: flex;
+  flex-direction: row;
+  width: 166px;
+  height: 16px;
+  justify-content: space-between;
+  align-content: center;
+  padding-bottom: 24px;
+}
```

Comment:
```suggestion
.rating {
  display: flex;
  justify-content: space-between;
  align-content: center;
  padding-bottom: 24px;
}
```

Reply:
also center stars horizontally with adding flex on rating__stars

Reply:
if I delete width than space-between doesn't work, how can i change it?